pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -154,9 +161,18 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 		// cache may be stale and we don't want to overwrite a prior update
 		// to status with this stale state.
 	} else if _, err := c.updateStatus(pr); err != nil {
-		c.Logger.Warn("Failed to update PipelineRun status", zap.Error(err))
+		message := "Failed to update PipelineRun status %s"
+		c.Logger.Warnf(message, zap.Error(err))
+		c.Recorder.Eventf(pr, corev1.EventTypeWarning, eventReasonFailed, message, zap.Error(err))
 		return err
 	}
+
+	if err == nil {
+		c.Recorder.Eventf(pr, corev1.EventTypeNormal, eventReasonSucceeded, "PipelineRun '5s' is done.", pr.Name)
```

Comment:
Typo: I think you meant `PipelineRun %q is done.` :)

Reply:
I think done is misleading word here. It could imply that pipelinerun is completed but technically this event should imply reconciliation is completed successfully. So probably a better msg would be `Pipelinerun successfully reconciled` (?)

Reply:
Good catch.  :-)

Agreed on changing the success message.